legal-api/tests/unit/resources/v1/test_business_filings/test_filings.py

```diff
@@ -110,6 +113,68 @@ def test_get_one_business_filing_by_id(session, client, jwt):
     assert rv.json['filing']['business'] == ANNUAL_REPORT['filing']['business']
 
 
+def test_business_filing_ia_parties(session, client, jwt):
+    """Assert that the business info cannot be received in a valid JSONSchema format."""
```

Comment:
"Can" be received?

Reply:
Yes... Will change the comments. Thanks.